app/javascript/alchemy_admin/i18n.js

```diff
@@ -36,7 +29,20 @@ function getTranslation(key) {
   return translations[key] || key
 }
 
-export default function translate(key, replacement) {
+export function currentLocale() {
+  if (document.documentElement.lang) {
+    return document.documentElement.lang
+  }
+  if (Alchemy.locale === undefined || Alchemy.locale === null) {
```

Comment:
I think we can now remove this, because we now have the locale on the document, right? We could still warn if this is not preset.  But checking the global Alchemy object should not be necessary anymore.